build: Introduce riscv default configuration

RuoqingHe opened the pull request:
Add riscv `ch_defconfig` file for kernel used by cloud-hypervisor.

Signed-off-by: Ruoqing He <heruoqing@iscas.ac.cn>

rbradford commented:
How about extending the workflows to cross compile this?

rbradford reviewed (approved):
Update this with the changes in #17 and then merge?

RuoqingHe commented:
> Update this with the changes in #17 and then merge?

Could we merge #17 first and then I make changes accordingly here 🙂

RuoqingHe committed:
build: Introduce riscv default configuration

Add riscv `ch_defconfig` file for kernel used by cloud-hypervisor.

Signed-off-by: Ruoqing He <heruoqing@iscas.ac.cn>

RuoqingHe committed:
github: Add riscv kernel build and upload

Add required pacakge for cross-compile riscv kernel, workflow for build
and upload kernel.

Signed-off-by: Ruoqing He <heruoqing@iscas.ac.cn>
